hh/actions/university_data.yaml

```diff
@@ -977,6 +977,10 @@ wc:
   - Woodstock
   BHC School of Design:
   - Cape Town
+  Boston Media House:
+  - Durban
+  - Pretoria
+  - Sandton
```

Comment:
If we've got these from new files, maybe run the import script again, to see if there's anything else that has changed? (Unless you did that already to add these, then just ignore this)